nvim/after/ftplugin/tex.lua

```diff
@@ -1,23 +1,19 @@
-local M = {}
-
-local config = require("nvim-surround.config")
-
 require("nvim-surround").buffer_setup({
   surrounds = {
     ["e"] = {
       add = function()
-        local env = require("nvim-surround.config").get_input "Environment: "
+        local env = require("nvim-surround.config").get_input("Environment: ")
         return { { "\\begin{" .. env .. "}" }, { "\\end{" .. env .. "}" } }
       end,
     },
     ["Q"] = {
       add = { "``", "''" },
-      find = "%b``''",
```

Comment:
The `%b[c1][c2]` pattern matches a balanced set of `c1` and `c2` characters (where `c1 != c2`). It's often used for matching parentheses or braces. In this case it won't work, since the left/right pair are multiple characters.